ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot

```diff
@@ -67,11 +77,10 @@ Verify Upload Download In Data Science Pipelines Using The kfp Python Package
     ...    pipeline_params=${upload_download_dict}
     [Teardown]    Projects.Delete Project Via CLI By Display Name    ${PROJECT_NAME}
 
-
 Verify Ods Users Can Create And Run A Data Science Pipeline With Ray Using The kfp Python Package
     [Documentation]    Creates, runs pipelines with regular user. Double check the pipeline result and clean
     ...    the pipeline resources.
-    [Tags]      Tier1
+    [Tags]      Tier1    ExcludeOnDisconnected
```

Comment:
Why Excluded from disconnected?

Reply:
This test was failing on disconnected

Reply:
Is it an automation issue or is it an issue with ray and an airgap environment ?

Reply:
I believe it's an automation bug. The image used for the test should be mirrored. I'll write you about this on slack

Reply:
Please do not exclude it. Rather let's fix it ;-)

Reply:
I've added it back and I'll create a Jira about fixing it when possible